beerus_core/src/lightclient/ethereum/helios_lightclient.rs

```diff
@@ -48,6 +48,14 @@ impl EthereumLightClient for HeliosLightClient {
     async fn get_block_number(&self) -> eyre::Result<u64> {
         self.helios_light_client.get_block_number().await
     }
+
+    async fn get_code(
+        &self,
+        address: &ethers::types::Address,
```

Comment:
```suggestion
        address: &Address,
```
import `ethers::types::Address`